task/03-date-tasks.js

```diff
@@ -76,14 +78,15 @@ function isLeapYear(date) {
  *    Date(2000,1,1,10,0,0),  Date(2000,1,1,15,20,10,453)   => "05:20:10.453"
  */
 function timeSpanToString(startDate, endDate) {
-   throw new Error('Not implemented');
+    let time = new Date(endDate - startDate);
+    return `${parseInt((endDate.getTime() - startDate.getTime()) / (1000 * 60 * 60)) < 10 ? '0' + parseInt((endDate.getTime() - startDate.getTime()) / (1000 * 60 * 60)) : parseInt((endDate.getTime() - startDate.getTime()) / (1000 * 60 * 60))}:${time.getMinutes() < 10 ? '0' + time.getMinutes() : time.getMinutes()}:${time.getSeconds() < 10 ? '0' + time.getSeconds() : time.getSeconds()}.${time.getMilliseconds() < 10 ? '00' + time.getMilliseconds() : time.getMilliseconds() < 100 ? '0' + time.getMilliseconds() : time.getMilliseconds()}`;
 }
```

Comment:
avoid this code style, if-else is more readable than multiple conditional operators ? :